src/lamplib/src/genny/templates/documentation.md.j2

```diff
@@ -1,6 +1,6 @@
-# Workload Documenation
+# Workload Documentation 
```

Comment:
The added space here makes me think that we aren't linting this. That's probably okay in the commit queue, but running some linter over this locally might be a good idea.